assets/scss/main.scss

```diff
@@ -5,9 +5,9 @@
 
 @import "../vendor/bootstrap/scss/bootstrap";
 
-@import "../vendor/Font-Awesome//web-fonts-with-css/scss/fontawesome.scss";
-@import "../vendor/Font-Awesome//web-fonts-with-css/scss/fa-solid.scss";
-@import "../vendor/Font-Awesome//web-fonts-with-css/scss/fa-brands.scss";
+@import "../vendor/Font-Awesome/scss/fontawesome.scss";
+@import "../vendor/Font-Awesome/scss/solid.scss";
+@import "../vendor/Font-Awesome/scss/brands.scss";
```

Comment:
https://github.com/FortAwesome/Font-Awesome/blob/master/UPGRADING.md gave the hints, particularly:

[5.1.0->5.1.1](https://github.com/FortAwesome/Font-Awesome/blob/master/UPGRADING.md#510-to-511)
[5.5.0-5.6.0](https://github.com/FortAwesome/Font-Awesome/blob/master/UPGRADING.md#550-to-560)